pro/src/screens/OfferIndividual/PriceCategories/PriceCategories.tsx

```diff
@@ -39,18 +47,31 @@ const PriceCategories = ({ offer }: IPriceCategories): JSX.Element => {
     if (isSubmittingFromRouteLeavingGuard) {
       return
     }
-    navigate(
-      getOfferIndividualUrl({
-        offerId: offer.id,
-        step:
-          mode === OFFER_WIZARD_MODE.EDITION
-            ? OFFER_WIZARD_STEP_IDS.SUMMARY
-            : isClickingDraft
-            ? OFFER_WIZARD_STEP_IDS.TARIFS
-            : OFFER_WIZARD_STEP_IDS.STOCKS,
-        mode,
-      })
-    )
+    const afterSubmitUrl = getOfferIndividualUrl({
+      offerId: offer.id,
+      step:
+        mode === OFFER_WIZARD_MODE.EDITION
+          ? OFFER_WIZARD_STEP_IDS.SUMMARY
+          : isClickingDraft
+          ? OFFER_WIZARD_STEP_IDS.TARIFS
+          : OFFER_WIZARD_STEP_IDS.STOCKS,
+      mode,
+    })
+    logEvent?.(Events.CLICKED_OFFER_FORM_NAVIGATION, {
+      from: OFFER_WIZARD_STEP_IDS.TARIFS,
```

Comment:
Ok sur le principe pour respecter le pattern actuel mais je me demande pourquoi on envoie autant d'infos dans les events, surtout que certaines ont l'air redondantes.

- Pour "from" et "to", est-ce que ce n'est pas une info qui est déjà trackée via les changements de page du router ? Si le tracking de pages marche bien normalement on a pas besoin de le repréciser
- Pour "used" j'ai l'impression que ça deviendra redondant avec isEdition/isDraft quand le route leaving guard ne permettra plus de soumettre

Reply:
ça a été fait avec la data mais en gros
- je crois que le from to leur facilite la vie
- les used ne recoupe pas exactement les boutons 

(il ya plein de choses qu'on pourrait améliorer mais je pense qu'ils aiment la stabilité côté data :)